core/src/main/scala/org/ensime/core/Project.scala

```diff
@@ -62,7 +65,7 @@ class Project(
     // make sure the "reliable" dependencies are cleaned up
     Try(classfileWatcher.shutdown())
     Try(sourceWatcher.shutdown())
-    Try(searchService.shutdown())
+    Try(Await.ready(searchService.shutdown(), DB_SHUTDOWN_TIMEOUT))
```

Comment:
What do you suggest concretely? What does not wait mean? 0 seconds? That's just like skipping the whole call and not shut Slick down. Not sure what the implications are. How about reducing it to 5 seconds or something? Not sure if this ever takes longer than a split second anyways, but better to have a timeout instead of Duration.Inf just in case.